erpnext/stock/doctype/delivery_note/test_delivery_note.py

```diff
@@ -170,6 +170,10 @@ def test_serialized(self):
 			"delivery_document_no": dn.name
 		})
 
+		si = make_sales_invoice(dn.name)
+		si.insert(ignore_permissions=True)
+		self.assertEquals(dn.items[0].serial_no, si.items[0].serial_no)
+
```

Comment:
write a test case for partial invoicing and if number of serial nos does not match with qty